src/components/EntitlementsAgreementsList/EntitlementAgreementsList.js

```diff
@@ -92,6 +94,9 @@ const EntitlementAgreementsList = (
         id={id}
         interactive={false}
         isEmptyMessage={isEmptyMessage}
+        onNeedMoreData={onNeedMoreData}
```

Comment:
same comment as above. See how the MCL prop and the handler prop have the same name. Lets avoid that.